### DISCUSSION TITLE
question  how to select a group of recipes for a week, maybe more than for all days  and select for a day from there

### DISCUSSION THREAD

**Discussion by sdetweil** [mealie-recipes/mealie#3302] [General]
today we decide on the recipes on Friday, and write them on a pad 

then cross them off as we make them during the week. 

sometimes we roll the extras, or one or two to the next week

really week planning instead of daily

is it possible to do today?

**Comment by michael-genson** [comment]
It sounds like you've checked out the meal planner, based on the issue you raised in https://github.com/mealie-recipes/mealie/issues/3293. This is exactly what the meal planner was designed to do :) I will take a look at that issue report in a bit

Currently the meal planner is designed as a daily planner, rather than weekly, though I personally operate similarly (planning for the week and figuring out when I'm cooking later). Currently this isn't supported, but possibly will be in the future

**Comment by andrew-codechimp** [comment]
This sounds similar to a FR I was about to raise.
I plan my meal plan for at least 7 days ahead.  Currently the meal planner defaults to the next 7 days including today so I always have to go and change the calendar to increase the range when editing/viewing what's ahead.
It would be good to have either a user definable default range, remember it for a user or maybe some clever scrolling where it fills the available space on the screen and loads more as you scroll.

**Reply by boc-the-git** [reply]
Ask and ye shall receive (~10 days ago)!
https://github.com/mealie-recipes/mealie/pull/3650
